tests/src/Kernel/Entity/OgMembershipTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\OgMembershipTest.
+ */
```

Comment:
Heads-up: the coding standards have changes regarding this docblock, it is no longer necessary for namespaced classes. See [issue #2304909: Relax requirement for @file when using OO Class or Interface per file](https://www.drupal.org/node/2304909).